examples/fluid_incompressible/benchmark/BrinkmanChannel2D/TRT/params.lua

```diff
@@ -0,0 +1,46 @@
+require "args"
```

Comment:
You can access shell environment variables in Lua:

```suggestion
F0 = os.getenv("F0")
```